index.js

```diff
@@ -104,6 +104,31 @@ class ServerlessAppsyncPlugin {
     return { "Fn::GetAtt": [lambdaLogicalId, "Arn"] };
   }
 
+  getDbClusterArn(config) {
+    if (config && config.dbClusterIdentifier) {
+      return this.generateDbClusterArn(config.dbClusterIdentifier, config.region);
```

Comment:
@roznalex Could we maybe implement it so that:
- if an arn is given, use it
- if an "identifier" is given, generate the Arn from it

I would also keep an eye on Cloudformation's doc. I wouldn't be surprised if they'd "fix it" or add a new key `dbClusterArn`

Reply:
@bboure sure. So should we add support for two options: dbClusterIdentifier and dbClusterArn?

Reply:
No, I would wait and see what AWS does.
On second thought, let's leave it as it is and keep it "in sync" with cloudformations

Reply:
@roznalex ok, so in fact that means that `dbClusterIdentifier` should receive an arn, and we could forget about generating it
What do you think?

Reply:
@bboure I'm not sure that it is the right decision, because a user needs to build this ARN by himself. According to [documentation](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-rds-dbcluster.html) Ref function returns a resource name.

Reply:
There is the `Fn::GetAtt` function, but now I see that it does not return the Arn (not sure).
Let's keep it as it is. It would be good to maybe write a post on AWS forum and ask for more info on this.